### DISCUSSION TITLE
Can't work out why ddns-updater container is unhealthy

### DISCUSSION THREAD

**Discussion by mpfl** [qdm12/ddns-updater#501] [Q&A] [answered]
Fired up ddns-updater today for my proxied Cloudflare setup. The container is reporting as unhealthy but I can't find any issues in the logs. Changed log level to debug and still couldn't find anything there.

How can I start to diagnose the issue?

**Comment by qdm12** [comment]
What are you using to run the container?

Also what's the healthcheck defined when you `docker inspect ddns-updater`?

**Reply by mpfl** [reply, marked as answer]
Aha! `docker inspect` gave me the information I needed to go back to the documentation and realise that I had left the vital `"proxied": true` from my config.

Problem existed between keyboard and chair.

Thanks for the tips! That's what I get for not RTFM.
